docs/accelerometer.rst

```diff
@@ -3,7 +3,10 @@ Accelerometer
 
 .. py:module:: microbit.accelerometer
 
-This module gives you access to the on-board accelerometer.
+This module gives you access to the on-board accelerometer. As a convenience
```

Comment:
I wonder if we should be lying about this being a module. I initially wrote it like that, because I thought they were modules, but they really are not.